script.js

```diff
@@ -13,3 +13,203 @@ navLinks.forEach((link) => {
     mobileIcon.classList.remove('active');
   });
 });
+const data = [
+  {
+    imgSrc: './Asset/nasir.svg',
+    title: 'Tonic',
+    canopy: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    languages: ['HTML', 'CSS', 'JavaScript'],
+  },
+  {
+    imgSrc: './Asset/avail.svg',
+    title: 'Multi-Post stories',
+    canopy: 'CANOPY',
+    role: 'Back End Dev',
+    year: '2015',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    languages: ['HTML', 'CSS', 'JavaScript'],
+  },
```

Comment:
As requested by the previous reviewer, we need to update the main page so that the projects section is created dynamically using the information stored in the JavaScript array of objects `const data [ { . . }, { . . . }, . . . ]`. 
Remember, all of the HTML in that section is created when the page loads.

After that, you could implement the popup window. please kindly remove all the hard-coded markup in the work section and let's populate the section with the data from the Javascript.

👇 👇 
![image](https://user-images.githubusercontent.com/78251927/232159237-e95108f5-a0b4-40d8-ac3b-57caca1c9f97.png)